### PR TITLE
Tree_calculation

### PR DESCRIPTION
**Pull request by Sravya12379**

### CODE REVIEW

File: functions/tree_calculation.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+from colorama import Fore
+colorama.init()
+
+def calc_trees(offset):
+    """1 gram of carbon produces 3.67grams of co2
+       1 tree can absorb 21,772.416 grams of co2 in its life time on an avg
+       :param offset
+       :return %of tree saved """
+    co2_prod= 3.67*offset
+    Tree_saved=(co2_prod/(21772.416))*100
```
Comment (robynfsj) [inline functions/tree_calculation.py]: We will need to change this variable name to all lower case. Anete said we need to make sure our code follows PEP 8. [PEP 8 says variable names need to be in lower case.](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names). 

I think a better name would be tree_equivalent or carb_offset_tree_equivalent as we are technically not saving trees, but we are saying that the carbon a user has saved by walking is the equivalent that would be offset by planting this many trees.

**Comment by clemcodes** [comment]
The calculation is correct, brilliant! I like that you use the color highlight in the console.
I just neatened up a bit, renamed the variable to increase the readability, and included situation where the result is more than more than or equal to 100%.

File: functions/tree_calculation.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        trees = str(format(num_of_trees * 100, ".2f") + '% of a tree')
+    elif num_of_trees == 1:
+        trees = '1 tree'
+    else:
+        trees = f'{int(num_of_trees)} trees'
+
+    print(Fore.GREEN
+          + "Your carbon offset amounted to "
+          + trees +
+          " being planted 🌳🌳🌳 Good job!")
```
Comment (clemcodes) [inline functions/tree_calculation.py:29]: Can move print lines to the main script later if we just want the function to focus on the result

File: functions/tree_calculation.py [hunk 3]
```diff
@@ -0,0 +1,41 @@
+import colorama
+from colorama import Fore
+colorama.init()
```
Comment (clemcodes) [inline functions/tree_calculation.py:3]: These are libraries print colored terminal text, can move with the print lines to the script later too if needed

**Review by clemcodes — APPROVED** [review]

File: functions/tree_calculation.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
+import colorama
+from colorama import Fore
```
Comment (robynfsj) [inline functions/tree_calculation.py:2]: This is already imported when you imported colorama in line 1.

**Review by robynfsj — APPROVED** [review]
Brill! Just some minor changes needed but we can look at them later.

**Review by robynfsj — APPROVED** [review]
Brill! Just some minor changes needed but we can sort those when we add it into the main program.

File: functions/tree_calculation.py (hunk 1 above)
Comment (Sravya12379) [reply functions/tree_calculation.py]: you are right!! I'm gonna change and update

File: functions/tree_calculation.py [hunk 5]
```diff
@@ -0,0 +1,22 @@
+import colorama
+from colorama import Fore
+colorama.init()
+
+def calc_trees(offset):
+    """1 gram of carbon produces 3.67grams of co2
```
[the remark this answers is not in the document]
Comment (Sravya12379) [reply functions/tree_calculation.py]: Ok!! I will upload the reference then

[the remark this answers is not in the document]
Comment (robynfsj) [reply functions/tree_calculation.py]: It doesn't need to go in the code, just in the report 🙂

**Review by rajwinderb — APPROVED** [review]
Looks fab! Agree with the rest of the comments 😄

File: functions/tree_calculation.py (hunk 1 above)
Comment (clemcodes) [reply functions/tree_calculation.py]: That's the initial draft, I renamed variables and added extra conditions, please go to file changed tab to review the final changed files before requesting or making any changes @robynfsj @Sravya12379

File: functions/tree_calculation.py (hunk 4 above)
Comment (clemcodes) [reply functions/tree_calculation.py:2]: I just tried to remove it, but got error `NameError: name 'colorama' is not defined`, wouldn't work without either of them unfortunately :'(

File: functions/tree_calculation.py (hunk 1 above)
Comment (clemcodes) [reply functions/tree_calculation.py]: oh, I pushed another commit that renamed variable names earlier, could you have a look at that one? here is the link: https://github.com/Walk2Zero/walk2zero/pull/32/commits/e680772f0e3a224a5875241e9032d7e7e224d89a

**Pull request merged**